Controller.php

```diff
@@ -124,7 +124,13 @@ public function index($errorMessage = false)
             ]);
         }
 
-        $isClientConfigurable = StaticContainer::get('GoogleAnalyticsImporter.isClientConfigurable');
+        if ($isConnectAccountsActivated) {
+            $isClientConfigurable = StaticContainer::get('GoogleAnalyticsImporter.isClientConfigurable');
+            $notification = new Notification(Piwik::translate('GoogleAnalyticsImporter_GoogleOauthCompleteWarning', ['<strong>', '</strong>']));
+            $notification->context = Notification::CONTEXT_WARNING;
+            $notification->raw = true;
+            Notification\Manager::notify('GoogleAnalyticsImporter_OauthCompletionWarning', $notification);
+        }
```

Comment:
It would be good to initialise the variable like `$isClientConfigurable = false;` above the if statement so that the view isn't being passed an undefined variable if `$isConnectAccountsActivated` is false. That would avoid the warning:
![image](https://github.com/matomo-org/plugin-GoogleAnalyticsImporter/assets/11394099/bc3ea0f8-4732-4b35-9f18-fb2dc5ce5696)

Reply:
@snake14 my bad, updated the code